app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Quran interpretation data from KSU-Electronic Mosshaf project"
+DESCRIPTION="Ayat data is use by other project like elforkane and mtafsir \
+get from original project to minimize maintenance .
+contain tafsir database:
+ * ar_muyassar.db
+ * sa3dy.db
+ * baghawy.db
+ * katheer.db
+ * qortoby.db
+ * tabary.db
+ * tafheem.db
+ * waseet.db
+ * tanweer.db
+ * russian.db
+ * e3rab.db
+ * indonesian.db
+ * tafheem.db
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+           2008-2016 Tanzil.net team
+		   "
```

Comment:
Move double quote and use tab not spaces:
```diff
-           2008-2016 Tanzil.net team
-		   "
+	2008-2016 Tanzil.net team"
```

Reply:
ok done .. thanks @Begasus

Reply:
> @mazbrili any update here?

.. yeah still investigate main application (elforkane) that have issue on close..
i hope i can find solution this week.. hopely

Reply:
You are still using spaces to start the line, please use a single tab instead